Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM continuumio/miniconda3:4.9.2-alpine
```

Comment:
Could these docker files go in a different directory from the main project directory? I'd prefer not to have tool-specific files up at the top level. Thanks!

Reply:
Hi Brandon. Sure I will move both docker related files to build directory.

Reply:
@brandon-rhodes  I have a problem. When I add the docker related files the exercise files are not accessible. I tried many things:
- changing dockerfile copy path
- changing context in build stage in docker-compose
with no luck.

You said that you like to move the build files to other directories I would like to know how you would fix this?

Reply:
<img width="1238" alt="current copied files" src="https://user-images.githubusercontent.com/11177270/127475976-2ac72955-bec9-411a-bcf3-c436419e5110.png">

Reply:
For the XEphem project, I have a directory `Containers` with the various Docker files inside — which lets me maintain several Docker files targeting different container operating systems.

https://github.com/XEphem/XEphem

Could you see whether a similar approach would work here?